Ginger/GingerCore/Actions/Communication/ActeMail.cs

```diff
@@ -216,7 +216,14 @@ public override void Execute()
             e.MailFrom = this.GetInputParamCalculatedValue(nameof(MailFrom));
             e.MailTo = this.GetInputParamCalculatedValue(nameof(Mailto));
             e.MailCC = this.GetInputParamCalculatedValue(nameof(Mailcc));
-            e.Attachments.Add(this.GetInputParamCalculatedValue(nameof(AttachmentFileName)));
+
+            //add multi attachment files
+            String[] fileslist = this.GetInputParamCalculatedValue(nameof(AttachmentFileName)).Split(new char[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6397283)